setup.py

```diff
@@ -50,8 +50,12 @@
             'pytest>=2.7',
             'requests-mock',
             'xmltodict',
+            'shopify_python==0.1.2',
         ],
     },
+    dependency_links=[
+        'git+https://github.com/Shopify/shopify_python.git@v0.1.2#egg=shopify_python-0.1.2',
```

Comment:
> syntactically incorrect

Right sorry it should have been `-e git+git://github.com` (but I did use _e.g._ 😛 )

If `dependency_links` is deprecated, let's switch to using separate requirements files soon (doens't have to be this PR).